tests/sandbox/test_jax.py

```diff
@@ -222,6 +222,21 @@ def test_jax_basic():
     )
 
 
+def test_jax_eye():
+    """Tests that jaxification of eye operator
+
+    Currently WIP
+    """
+
+    # eye = theano.tensor.eye(3).shape
+    x = tt.iscalar()
+    out = tt.eye(x)
+
+    out_fg = theano.gof.FunctionGraph([x], [out])
+
+    compare_jax_and_py(out_fg, [3.0])
```

Comment:
The numeric value for `x` should be a scalar **integer**, since that's what the symbolic version is (i.e. `tt.iscalar` means "integer scalar").